html/modules/custom/rwr_sitrep/rwr_sitrep.install

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * @file
+ * Update hooks.
+ */
+
+use Drupal\taxonomy\Entity\Term;
+use Drupal\taxonomy\Entity\Vocabulary;
+
+/**
+ * Add taxonomy term for sitreps and capture the tid.
+ */
+function rwr_sitrep_update_10300() {
+
+  // See if we're already set.
+  if (\Drupal::state()->get('rwr_sitrep:sitrep_tid')) {
+    \Drupal::logger('rwr_sitrep')->notice('Sitrep tid already set');
+    return;
+  }
+
+  // Ensure vocabulary exists.
+  $vocabularies = Vocabulary::loadMultiple();
+  if (!isset($vocabularies['cluster_or_working_group_type'])) {
+    $vocabulary = Vocabulary::create([
+      'vid' => 'cluster_or_working_group_type',
+      'name' => t('Cluster or Working Group Type'),
+      'description' => t('For connecting subgroups across Operations'),
+    ]);
+    \Drupal::logger('rwr_sitrep')->notice('Creating cluster or working group type vocabulary');
+    $vocabulary->save();
+  }
+  else {
+    \Drupal::logger('rwr_sitrep')->notice('Cluster or working group type vocabulary already exists');
+  }
+
+  // Ensure term exists.
+  $terms = \Drupal::entityTypeManager()->getStorage('taxonomy_term')->loadTree('cluster_or_working_group_type');
+  foreach ($terms as $term) {
+    if ($term->name == 'Situation Report') {
+      \Drupal::state()->set('rwr_sitrep:sitrep_tid', $term->tid);
+      \Drupal::logger('rwr_sitrep')->notice('Sitrep term already exists, saving tid');
+      return;
+    }
+  }
+  $term = Term::create([
+    'name' => 'Situation Report',
+    'vid' => 'cluster_or_working_group_type',
+  ]);
+  $term->save();
```

Comment:
Add a term for `Cluster` and `Working group` as well so they can be used on new sub groups.

Maybe also a term for `Sectors`, see sidebar on https://response.reliefweb.int/afghanistan